File: pl/gui/tk/hello/hello.pl
#!/usr/bin/perl -w
#
use Tk ;

# Main window.
my $mw     = new MainWindow() ;
my $label  = $mw->Label(-text=>"Hello, World!")->pack() ;
my $button = $mw->Button(
	-text    => "Quit",
	-command => sub{ exit }
)->pack() ;
 
MainLoop();

File: py/old/test/gui/tk/hello/hello.py
#!/usr/bin/python3

import tkinter

#handlers
def button_clicked():
    print ("Hello, World!")

root = tkinter.Tk ()

button = tkinter.Button (
        root,
        bg="red",
        text="It's clickable, really!",
        command=button_clicked
)
button.pack()


#it must be in the end
root.mainloop()
